src/components/TodoCard/TodoCard.jsx

```diff
@@ -26,18 +28,26 @@ function TodoCard({
         ' todo-card--deadline__danger'
         : '';
 
+    const toggleIsDone = () => {
+        todo.isDone = !isDone;
+        setIsDone(!isDone);
```

Comment:
Why do you duplicate this state? You should have single source of truth. It should come as props